README.md

```diff
@@ -1,37 +1,35 @@
 # Alacritty Theme
 
-[![license](https://img.shields.io/github/license/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/blob/master/LICENSE)
-[![GitHub tag](https://img.shields.io/github/tag/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/tags)
-[![GitHub last commit (branch)](https://img.shields.io/github/last-commit/eendroroy/alacritty-theme/master.svg)](https://github.com/eendroroy/alacritty-theme)
+Collection of colorschemes for easy configuration of the [Alacritty terminal
+emulator].
 
-[![GitHub issues](https://img.shields.io/github/issues/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/issues)
-[![GitHub closed issues](https://img.shields.io/github/issues-closed/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/issues?q=is%3Aissue+is%3Aclosed)
-[![GitHub pull requests](https://img.shields.io/github/issues-pr/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/pulls)
-[![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed/eendroroy/alacritty-theme.svg)](https://github.com/eendroroy/alacritty-theme/pulls?q=is%3Apr+is%3Aclosed)
```

Comment:
I think the badges themselves don't really provide any value, so it's much nicer to save that space and see more of the rest of the page.